openlibrary/templates/site/head.html

```diff
@@ -5,12 +5,14 @@
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <meta name="title" content="" />
-    <meta name = "keywords" content = "free books, books to read, free ebooks,audio books,read books for free, read books online, online library">
+    <meta name="keywords" content="free books, books to read, free ebooks, audio books, read books for free, read books online, online library">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="author" content="OpenLibrary.org" />
     <meta name="creator" content="OpenLibrary.org" />
     <meta name="copyright" content="Original content copyright; 2007-2015" />
     <meta name="distribution" content="Global" />
+    $if not request.canonical_url.startswith("https://openlibrary.org/"):
```

Comment:
TIL about the `request` global :D I did a search for `def canonical_url` in the codebase, and it looks like this is the corresponding class :) 

https://github.com/internetarchive/openlibrary/blob/1cf1b7791749754d1f114973d57b7fc65cfc3ec0/openlibrary/plugins/upstream/utils.py#L809-L815